requirements.yml

```diff
@@ -19,7 +19,7 @@
   version: 1.1.0
 
 - src: openmicroscopy.omero-server
-  version: 2.0.0-m2
+  version: 2.0.1
 
 - src: openmicroscopy.postgresql
-  version: 2.0.0
+  version: 2.0.1
```

Comment:
Should be `3.0.1` - https://galaxy.ansible.com/openmicroscopy/postgresql/ ?

Reply:
Yup. Thanks. (Feel free to push yourself)